### PR TITLE
LOGIN_REDIRECT_URL: new site configuration setting for LMS

### PR DESCRIPTION
**Pull request by OmarIthawi**
Jira: RED-1340

### CODE REVIEW

File: common/djangoapps/student/helpers.py [hunk 1]
```diff
@@ -285,7 +285,9 @@ def get_next_url_for_login_page(request):
     redirect_to = get_redirect_to(request)
     if not redirect_to:
         try:
-            redirect_to = reverse('dashboard')
+            default_redirect_to = reverse('dashboard')
+            site_redirect_to = configuration_helpers.get_value('LOGIN_REDIRECT_URL', reverse('dashboard'))
+            redirect_to = site_redirect_to or default_redirect_to  # Falsy site config values should not be used
```
Comment (johnbaldwin) [inline common/djangoapps/student/helpers.py:290]: Could we just do this with the following?

```
redirect_to = configuration_helpers.get_value('LOGIN_REDIRECT_URL', reverse('dashboard'))
```

Comment (OmarIthawi) [reply common/djangoapps/student/helpers.py:290]: Unfortunately not. AMC puts a `null` value by default, so it'll default to null.

I've added a test case for that just to ensure it works properly:

https://github.com/appsembler/edx-platform/blob/01f6c7a79ca8599c295564e6d558ee55b50cbc7e/common/djangoapps/student/tests/test_helpers.py#L144-L147

Comment (johnbaldwin) [reply common/djangoapps/student/helpers.py:290]: AMC puts a `null` value by default into `configuration_helpers.get_value('LOGIN_REDIRECT_URL)` ?

Comment (OmarIthawi) [reply common/djangoapps/student/helpers.py:290]: yes, `None` when read in Python or `null` in the json serialization.

I checked it yesterday:

<kbd>image</kbd>

**Review by thraxil — APPROVED** [review]

File: common/djangoapps/student/helpers.py (hunk 1 above)
Comment (johnbaldwin) [reply common/djangoapps/student/helpers.py:290]: What I'm seeing in the logic is the following without using intermediate variables

```
redirect_to = configuration_helpers.get_value('LOGIN_REDIRECT_URL', reverse('dashboard')) or reverse('dashboard')
```

**Review by johnbaldwin — APPROVED** [review]
Approving. My comment is just a minor one

File: common/djangoapps/student/helpers.py (hunk 1 above)
Comment (OmarIthawi) [reply common/djangoapps/student/helpers.py:290]: thanks @johnbaldwin! I won't modify it though

**Pull request merged**